src/elements/paginator/common/paginator-common.ts

```diff
@@ -18,14 +18,13 @@ export declare abstract class SbbPaginatorCommonElementMixinType {
   public length: number;
   public pageSize: number;
   public pageIndex: number;
-  public pagerPosition: 'start' | 'end';
+  public accessor pagerPosition: 'start' | 'end';
   public size: 'm' | 's';
```

Comment:
size should have accessor too (?)

Reply:
no idea :) i added it to be compliant with the mixin below, otherwise i had an error here